src/pages/RoomInvitePage.tsx

```diff
@@ -134,7 +136,7 @@ function RoomInvitePage({betas, report, policies}: RoomInvitePageProps) {
             if (isOptionInList) {
                 newSelectedOptions = selectedOptions.filter((selectedOption) => selectedOption.login !== option.login);
             } else {
-                newSelectedOptions = [...selectedOptions, {...option, isSelected: true}];
+                newSelectedOptions = [...selectedOptions, {...option, accountID: option.accountID ?? undefined, isSelected: true}];
```

Comment:
Is it possible to update `OptionsListUtils.MemberForList` accountID typing instead?

Reply:
I wanted to avoid changing other types, but apparently it didn't go as bad as I expected